sass/_footer.scss

```diff
@@ -1,32 +1,32 @@
 .site-footer {
-    color: #666;
-    @include rem(font-size, 13px);
-    padding: 30px 0;
-    
-    .site-info {
-        @extend %cf;
-        @include site-width;
-        
-        .site-credits {
-            float: left;
-        }
-        
-        .social-navigation {
-            float: right;
-            font-size: $font-size-normal;
-        }
-        
-        @media screen and (max-width: 405px) {
-            .site-credits,
-            .social-navigation {
-                float: none;
-                margin-top: 20px;
-                text-align: center;
-            }
-        }
+  color: #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets